packages/pyright-internal/src/tests/fourslash/importSuggestionWholeModuleImported.invokeCodeAction.fourslash.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="fourslash.ts" />
+
+// @filename: pyrightconfig.json
+//// {}
+
+// @filename: test.py
+//// import types
+//// foo: [|/*import*/|]TracebackType[|/*marker*/|]
+{
+    const importRange = helper.getPositionRange('import');
+
+    const codeActions = {
+        codeActions: [
+            {
+                title: `from types import TracebackType`,
+                edit: {
+                    changes: {
+                        'file:///test.py': [{ range: importRange, newText: 'types.' }],
+                    },
+                },
+                kind: 'quickfix',
+            },
+        ],
+    };
+
+    //@ts-expect-error https://github.com/DetachHead/basedpyright/issues/86
+    await helper.verifyCodeActions('included', {
+        noImportMarker: { codeActions: [] },
+        startMarker: codeActions,
```

Comment:
PEBKAC error

Reply:
really dumb that this fourslash thing doesn't complain on invalid markers here